nodebuilder/tests/swamp/swamp.go

```diff
@@ -69,15 +68,22 @@ func NewSwamp(t *testing.T, options ...Option) *Swamp {
 
 	// we create an arbitrary number of funded accounts
 	accounts := make([]string, 100)
-	for i := 0; i < 10; i++ {
-		accounts = append(accounts, tmrand.Str(9))
+	for i := 0; i < 100; i++ {
+		accounts[i] = tmrand.Str(9)
 	}
 
 	// TODO(@Bidon15): CoreClient(limitation)
```

Comment:
We can also remove this TODO. It's fine to make tested App assumption here and there is nothing TBD about this